packages/nimble-components/src/toggle-button/styles.ts

```diff
@@ -5,87 +5,106 @@ import {
     borderHoverColor,
     borderWidth,
     fillSelectedColor,
+    fillSelectedRgbPartialColor,
     smallDelay
 } from '../theme-provider/design-tokens';
 import { styles as buttonStyles } from '../patterns/button/styles';
 
 export const styles = css`
     ${buttonStyles}
 
-    .control[aria-pressed='true'] {
-        background-color: ${fillSelectedColor};
-        border-color: ${fillSelectedColor};
-        position: relative;
-        transition: box-shadow ${smallDelay} ease-in-out,
-            border-color ${smallDelay} ease-in-out,
-            background-size ${smallDelay} ease-in-out;
-        background-size: 100% 100%;
-        background-repeat: no-repeat;
-        background-position: center;
-    }
+    @layer default {
+        .control[aria-pressed='true'] {
+            background-color: transparent;
+            background-image: linear-gradient(
+                ${fillSelectedColor},
+                ${fillSelectedColor}
+            );
+            border-color: rgba(${fillSelectedRgbPartialColor}, 0.3);
+            position: relative;
+            transition: box-shadow ${smallDelay} ease-in-out,
+                border-color ${smallDelay} ease-in-out,
+                background-size ${smallDelay} ease-in-out;
+            background-size: 100% 100%;
+            background-repeat: no-repeat;
+            background-position: center;
```

Comment:
none of these are necessary to redefine right? These were all set in the default / base of `.control` in the buttonStyles already this is just overwriting everything with the same value?